compute_tools/src/lib.rs

```diff
@@ -21,6 +21,7 @@ pub mod metrics;
 mod migration;
 pub mod monitor;
 pub mod params;
+pub mod pg_bouncer;
```

Comment:
One last comment. It is `pgbouncer`. That's the name of the binary. Using the correct name makes it easier to search and find references.